apps/backend/minecraft_manager/host_agent.py

```
@@ -0,0 +1,344 @@
+    # ------------------------------------------------------------------
+
+    def status(self) -> dict[str, Any]:
+        """Return container-style status by probing the agent's health endpoint.
+
+        A successful GET /v1/health response is mapped to an "online" / "running"
+        state.  Any transport failure or non-200 response is mapped to "stopped"
+        without raising.
+        """
+        url = f"{self._base_url}/v1/health"
+        try:
+            code, body = self._client.request("GET", url, timeout=10.0)
```

Ruff (0.16.2)

[warning] Unpacked variable `body` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)